### ISSUE TITLE
[Question] Use a standard file name instead of `Title` Text field

### ISSUE THREAD

**Issue by FelipeRearden** [mo-seph/obsidian-note-from-template#10]
Hello @mo-seph !!!!!

Sorry if is a dumb question :)

Is it possible to define a specific file name in the template note in a way that when I’m creating a new note from template I could skip the text field `Tilte: [      ]`

I was trying to use: 

`template-name: filename`
`template-name: filename.md`
`template-name: ”filename”`
`template-name: ”filename.md”`

But I get nothing :(

Thanks ! 

Have a great day!

**Comment by mo-seph** [comment]
Not sure of the use case here - the title is used for the filename, so you wouldn't be able to use it as a template if it always made the same file. Can you tell me more? If it's to avoid creating lots of notes, there's a setting for that now.

**Comment by FelipeRearden** [comment]
> Can you tell me more? If it's to avoid creating lots of notes, there's a setting for that now.

Hello @mo-seph !!!!

Sorry for my bad explanation, let me rephrase !


My idea: 

I want to create a template with a predefined filename to new note created by the template note. 
For example:
Using the template note called Template3 and I want to name my new note with the filename `airplane` and save it inside the folder called Folder1

In my mind this should be my yaml:
```
template-output: "Folder1"
template-name: "airplane"
```

When opening the modal, I was expecting that the field `Title:` would be filled with the word `airplane` 

Am I doing something wrong? 

---

I'm asking you this because I'm thinking about 2 FR related to this behavior. I wanted to check with you before creating these new items. 

---

Let me know of you need more information :)

Have a great day!

**Comment by mo-seph** [comment]
There's a new `template-filename` field in the YAML you can use to construct a filename. Note - if you were putting links into the replacement text (e.g. `[[{{title}}]]`, make sure to use the new tag `filename` instead, e.g. `[[{{filename}}]]`

**Comment by FelipeRearden** [comment]
Thank you very much !!!!!!!!

it´s amazing @mo-seph ! I loved the new creator modal interface :)

Working perfect on version '0.1.6' Obsidian '0.12.19'

Have a great day!

**Comment by FelipeRearden** [comment]
Related to #15
